tests/internal/test_packages.py

```diff
@@ -52,6 +52,8 @@ def test_get_distributions():
             importlib_pkgs.add("typing-extensions")
         elif pkg.name == "pkgutil_resolve_name" and "pkgutil-resolve-name" in pkg_resources_ws:
             importlib_pkgs.add("pkgutil-resolve-name")
+        elif pkg.name == "importlib_metadata" and "importlib-metadata" in pkg_resources_ws:
+            importlib_pkgs.add("importlib-metadata")
```

Comment:
Is it possible that we need the opposite as well for 2.7 - 2.9?
https://app.datadoghq.com/ci/test-runs?query=test_level%3Atest%20%40test.name%3Atest_get_distributions%20%40test.status%3Afail&agg_m=count&agg_m_source=base&agg_t=count&currentTab=overview&eventStack=&fromUser=false&index=citest&start=1718008804947&end=1718613604947&paused=false

Reply:
I think some of this will get resolved by: https://github.com/DataDog/dd-trace-py/pull/9515

? since we'll more consistently be reporting the distribution name, and not the module name?